lib/galaxy/version.py

```diff
@@ -1,3 +1,3 @@
 VERSION_MAJOR = "21.01"
-VERSION_MINOR = None
+VERSION_MINOR = ""
```

Comment:
Some of the release tooling expects `None` for the `.0` release, so this change breaks that a bit. However we don't currently make point releases and I am overhauling the process, so I can incorporate this change if necessary.

Reply:
Yes, please, it should be possible to replace `foo is None` with `not foo` or similarly, I suppose.

Reply:
I assume I should also make it so that when we create future .0 releases, we use the empty string instead of `None`?

Reply:
Yep, we need to keep mypy happy :)